scripts/data/block_filter.jq

```diff
@@ -63,14 +63,32 @@ def block:
    }"
 ;
 
+def chain_state:
```

Comment:
I think you need to put it to a separate jq and add extra request in the get_block.sh otherwise how can you obtain prev_block?

Reply:
I dont get this.

Reply:
I think this is the only requirement left... For the alternative option if I wanted to create a chain_state with default values, I should do that in the `block_filter.jq`?